staging/src/k8s.io/client-go/README.md

```diff
@@ -18,7 +18,17 @@ client-go has the same release cycle as the Kubernetes main repository. For exam
 
 ### How to get it
 
-You can `go get` a release of client-go, e.g., `go get k8s.io/client-go/1.4/...` for the whole thing or `go get k8s.io/client-go/1.4/kubernetes` if you just need the clientset.
+You can `go get` to get a release of client-go, e.g., `go get k8s.io/client-go/1.4/...` or `go get k8s.io/client-go/1.4/kubernetes`.
+
+### How to use it
+
+Please refer to the [examples](example/).
+
+### Dependency management
+
+If your application depends on a package that client-go depends on, and you let the Go compiler find the dependency in `GOPATH`, you will end up with duplicated dependencies: one copy from the `GOPATH`, and one from the vendor folder of client-go. This will cause unexpected runtime error like flag redefinition. If this happens, you can either
```

Comment:
> This will cause unexpected runtime error like flag redefinition

This might cause unexpected runtime errors like flag redefinition, since the go compiler ends up importing both packages separately, even if they are exactly the same thing.